### DISCUSSION TITLE
Bioconda integration

### DISCUSSION THREAD

**Discussion by chrarnold** [churchmanlab/genewalk#58] [Q&A] [answered]
GeneWalker fits very well into Bioconda, do you have plans of adding it to Bioconda as well so that it can be installed via "conda install" also? Would be great, the tools looks very promising!

**Comment by ri23** [comment]
Thank you for the suggestion @chrarnold, sounds good, we will look into adding GeneWalk to Bioconda.

**Comment by bgyori** [comment]
Working on it, https://github.com/bioconda/bioconda-recipes/pull/17469, however, we need to add various dependencies to BioConda first for it to go through.

**Comment by bgyori** [comment]
It is getting very complicated with dependencies now 4 levels deep needing to be added to BioConda one by one, prospects aren't great of this happening soon.

**Comment by chrarnold** [comment]
Wow ok I did not know this is so complicated... Was setting up pip much easier? Thanks for the effort, I was under the assumption that adding a recipe to Bioconda is straight-forward?

**Comment by mfansler** [comment]
@chrarnold I made [some YAMLs that can be used to install GeneWalk via Conda](https://gist.github.com/mfansler/1821b5164de6ef9a20c75e63654a0c7d), e.g., 

    conda env create -f genewalk-latest.yaml

Basically, I install all dependencies that are already available through an Anaconda Cloud channel (Bioconda, Conda Forge, or Anaconda) and then let `pip` do the rest. Just be aware that best practice is that one should treat such an environment (Conda + Pip mixed) as immutable. If you want to change something in it, edit the YAML and recreate the env.

For the `genewalk-latest.yaml`, you may need to adjust the `ssh` to `https`, depending on how your GitHub access is configured.

**Comment by bgyori** [comment, marked as answer]
GeneWalk is now integrated with BioConda: https://anaconda.org/bioconda/genewalk (also see https://github.com/bioconda/bioconda-recipes/pull/17469).
